submissions/7SergSmith7/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>HTML-CSS-Popup</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav class="right_navigation">
+          <a href="#" class="right_navigation__mail">Mail</a>
+          <a href="#" class="right_navigation__images">Images</a>
+          <input type="checkbox" id="popup-check" class="popup_btn" />
+          <label class="popup_btn-icon" tabindex="0" for="popup-check"></label>
```

Comment:
Attribute `tabindex` better add to non-interactive elements, or when you need to change the order of interactive elements.
No need to adding `tabindex` to the interactive elements without changing markup order.